package.json

```diff
@@ -65,7 +65,7 @@
     "choo": "^3.2.0",
     "collect-stream": "^1.1.1",
     "dat-design": "^1.4.2",
-    "dat-encoding": "^3.0.0",
+    "dat-key-as": "^1.0.0",
```

Comment:
i thought about adding this functionality to dat-encoding. should we rather add it there than making more modules, although dat-encoding doesn't do much yet?